src/app/services/security.service.ts

```diff
@@ -0,0 +1,57 @@
+import { AppSecurity } from '@aerogear/security';
+import { Injectable } from '@angular/core';
+import { OpenShiftService } from './openshift.service';
+import { VoyagerService } from './sync/voyager.service';
+import { AuthContextProvider } from '@aerogear/voyager-client';
+import { Platform } from '@ionic/angular';
+import { getUrl } from '@ionic/angular/dist/directives/navigation/stack-utils';
+
+@Injectable({
+    providedIn: 'root'
+})
+/**
+ * Service provides Apollo Voyager client
+ */
+export class SecurityService {
+  public security: AppSecurity | undefined;
+  public initialized: Promise<boolean>;
+  private readonly aerogear: VoyagerService;
+
+  constructor(private openShift: OpenShiftService, public platform: Platform, aerogear: VoyagerService) {
+      if (this.isEnabled()) {
+          this.security = new AppSecurity(this.openShift.getConfig());
+          this.aerogear = aerogear;
+      }
+  }
+
+  getURL() {
+    const config = this.security.getConfig();
+    console.log(config);
+    return config;
+  }
+
+  clientInit() {
+    this.security.clientInit()
+     .then(clientInit => {
+       console.log('Client Data :', clientInit);
+       if (clientInit.data.disabled) {
+        alert('Security Warning :' + clientInit.data.disabledMessage);
```

Comment:
We can use ionic ToastController

Reply:
doesn't look the part https://ionicframework.com/docs/api/toast, maybe use could use https://ionicframework.com/docs/api/alert, but doesn't look much better that a standard alert